Storage/tests/Unit/Connection/RestTest.php

```diff
@@ -681,11 +681,11 @@ public function retryStrategyCases()
         foreach ($retryCases as $retryCase) {
             // For retry always
             $case = $retryCase;
-            $case[3]['retryStrategy'] = Rest::getStrategyAlwaysKey();
+            $case[3]['retryStrategy'] = 'always';
```

Comment:
I don't think we should be using strings like these.
Having these defined in a central place, makes sure we can change it in a place and it ensures it will change everywhere.

It also helps the user to use these values without having to _know_ what the exact values are.

I'd be open to change the location of these, if the trait doesn't seem the right place to put it.

Reply:
In fact, I'd be of the view of removing the prvt static properties from the `RetryTrait` to constants in `StorageClient` so that the users can easily access it.

With the current implementation, the user fetches these from the Rest class(which is supposed to be an abstraction) or just use strings(which might be easy, but not future proof).

Reply:
If this is something the customers will be using, then absolutely it should be a `const`. But if it's just a string being used in tests, hardcoding is okay. It sounds like it's the former, from your comment?